tests/lib/rules/all.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 const RuleTester = require('eslint').RuleTester;
 const assert = require('assert');
+const kebabCase = require('kebab-case');
```

Comment:
Thanks. I don't think `kebabCase` is used here. Once this is removed we can merge it :)

Reply:
done